### PR TITLE
Use LR schedule for beta and epsilon

### PR DESCRIPTION
**Pull request by andrewcoh**
### Proposed change(s)

This PR extends the schedule used to decay the learning rate to beta and epsilon.  Currently, beta and epsilon are decayed automatically and this is not configurable.  This impacts training scenarios like self-play which require some degree of exploration all through out an experiment as the opponent changes.

Additionally, this PR adds beta and epsilon to tensorboard to track these values.

### Types of change(s)

- [ ] Bug fix
- [x] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [x] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: ml-agents/mlagents/trainers/models.py [hunk 1]
```diff
@@ -69,17 +70,15 @@ def create_learning_rate(
...
+            parameter_rate = tf.train.polynomial_decay(
+                parameter, global_step, max_step, min_value, power=1.0
             )
         else:
-            raise UnityTrainerException(
-                "The learning rate schedule {} is invalid.".format(lr_schedule)
-            )
-        return learning_rate
+            raise UnityTrainerException("The schedule {} is invalid.".format(schedule))
+        return tf.stop_gradient(parameter_rate)
```
Comment (andrewcoh) [inline ml-agents/mlagents/trainers/models.py:81]: @ervteng added stop_gradient

File: ml-agents/mlagents/trainers/models.py [hunk 2]
```diff
@@ -69,17 +70,15 @@ def create_learning_rate(
         :param max_step: The maximum number of steps in the training run.
         :return: A Tensor containing the learning rate.
         """
-        if lr_schedule == LearningRateSchedule.CONSTANT:
-            learning_rate = tf.Variable(lr)
-        elif lr_schedule == LearningRateSchedule.LINEAR:
-            learning_rate = tf.train.polynomial_decay(
-                lr, global_step, max_step, 1e-10, power=1.0
+        if schedule == ScheduleType.CONSTANT:
+            parameter_rate = tf.Variable(parameter, trainable=False)
```
Comment (andrewcoh) [inline ml-agents/mlagents/trainers/models.py:74]: @ervteng added trainable=False
